receiver/kubeletstatsreceiver/internal/metadata/metrics.go

```diff
@@ -146,6 +147,7 @@ type VolumeMetrics struct {
 var K8sVolumeMetrics = VolumeMetrics{
 	Available:  (*MetricsBuilder).RecordK8sVolumeAvailableDataPoint,
 	Capacity:   (*MetricsBuilder).RecordK8sVolumeCapacityDataPoint,
+	Used:       (*MetricsBuilder).RecordK8sPodVolumeUsageDataPoint,
```

Comment:
Is this called at all 🤔 ? I think it should be used in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/internal/kubelet/volume.go#L16, right?

Reply:
good catch, thanks!